client/state/user-settings/README.md

```diff
@@ -0,0 +1,23 @@
+User Settings
+=============
+
+This store holds user settings from `/me/settings`.
+
+# Usage
+
+1. Render `QueryUserSettings` from `components/data/query-user-settings`
+2. Connect your component specifying proper setting names:
+```es6
```

Comment:
should this not be `js` here?

anyways, although we all know and love it as ES6 it's just JavaScript and the official name of the release is ES2015.